test/test_playbooks/tasks/job_template.yml

```diff
@@ -22,8 +22,10 @@
     server_url: "{{ foreman_server_url }}"
     verify_ssl: "{{ foreman_verify_ssl }}"
     name: "{{ job_template_name }}"
-    locations: "{{ job_template_locations }}"
-    organizations: "{{ job_template_organizations }}"
+    description_format: "{{ job_template_description_format | default(omit) }}"
+    job_category: "{{ job_template_category | default(omit) }}"
+    locations: "{{ job_template_locations | default(omit) }}"
+    organizations: "{{ job_template_organizations | default(omit) }}"
```

Comment:
I guess, the `| default(omit)` works only as expected, if you do not have a default value.